react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/GutenbergBridgeJS2Parent.java

```diff
@@ -17,10 +17,32 @@
         void onMediaFileUploadFailed(int mediaId);
     }
 
+    enum LogLevel {
```

Comment:
Let's add a line comment pointing back to RN's definition of the constants (https://github.com/facebook/react-native/blob/master/Libraries/polyfills/console.js#L376) so we know we're trying to mirror those, and hopefully catch a discrepancy in the future if the RN side constants change. What do you think?

Reply:
👍 a37fcf4